GameData/RP-1/Contracts/Earth Observation 1/EOSMagScan1.cfg

```diff
@@ -0,0 +1,127 @@
+CONTRACT_TYPE
+{
+	name = EOSMagScan1	
+	title = Magnetometry Satellite
+	group = EOS
+	agent = Federation Aeronautique Internationale
+	
+	description = <b>Program: Early Earth Observation Satellites<br>Type: <color=green>Required</color></b><br><br>Anyone that has ever used a compass is familiar with the magnetic fields that encapsulate Earth.  Launch a satellite to better charaterize the strength and direction of the magnetic field, and determine how far into space its influence is felt.&br;&br;<b><color="green">Make sure your craft contains a Magnetic Scan experiment.  Launch it into an elliptical orbit and return 1-days worth of science from both Low and High Earth Orbit.</color></b>&br;&br;Real life examples: IMP-A.
+
+	synopsis = Launch the first dedicated magnetometry satellite.
+
+	completedMessage = Our scientists now better understand the magnetosphere surrounding Earth and, combined with previous data, have determined the existance of a standing shock wave in the solar wind around the magnetic field.
+
+	sortKey = 402
+
+	cancellable = true
+	declinable = false
+	autoAccept = false
+	minExpiry = 0
+	maxExpiry = 0
+	maxCompletions = 1
+	maxSimultaneous = 1
+	deadline = 0
+
+	targetBody = HomeWorld()
+
+	// ************ REWARDS ************
+	prestige = Trivial       // 1.0x
+	advanceFunds = 0
+	rewardScience = 0
+	rewardFunds = 0
+	failureFunds = 0
+	rewardReputation = 60
+	failureReputation = 0 // was @rewardReputation
+
+	// ************ REQUIREMENTS ************
+
+	REQUIREMENT
+	{
+		name = ProgramActive
+		type = ProgramActive
+		program = EarlyEOS
+	}
+
+
+	PARAMETER
+	{
+		name = ScienceSatMagScan1
+		type = VesselParameterGroup
+		title = First magnetrometry satellite
+		define = ScienceSatMagScan1
+
+		PARAMETER
+		{
+			name = NewVessel
+			type = NewVessel
+			title = Launch a new vessel
+			hideChildren = true
+		}
+		PARAMETER 
+		{
+			name = Crewmembers
+			type = HasCrew
+			minCrew = 0
+			maxCrew = 0
+			title = Uncrewed
+			hideChildren = true
+		}
+
+		PARAMETER
+		{
+			name = OrbitSequence
+			type = Sequence
+			title = Survive in orbit for a day and transmit science
+
+			PARAMETER
+			{
+				name = HasPower
+				type = HasResource
+				resource = ElectricCharge
+				minQuantity = 1.0
+				title = Craft must have ElectricCharge after one day
+				hideChildren = true
+			}
+
+			PARAMETER
+			{
+				name = Orbit
+				type = Orbit
+				minApA = 36000000

```

Comment:
Same deal as Cherenkov, needs a max Pe (and probably a max Ap as well, something like 250Mm ?)